static/app/components/replays/replayController.tsx

```diff
@@ -40,10 +52,22 @@ function ReplayPlayPauseBar() {
       />
       <Button
         size="xsmall"
-        title={t('Go forward 10 seconds')}
-        icon={<IconClockwise size="sm" />}
-        onClick={() => setCurrentTime(currentTime + 10 * SECOND)}
-        aria-label={t('Go forward 10 seconds')}
+        title={t('Jump to next event')}
+        icon={<IconNext size="sm" />}
+        onClick={() => {
+          const startTimestamp = replay?.getEvent().startTimestamp || 0;
```

Comment:
startTimestamp === 0 should be invalid right? we could probably return if falsey?

Reply:
yeah, this can be simpler